### PR TITLE
Add support for swift in documentation

### PR DESCRIPTION
**Pull request by ppult-gini**
Add support for swift and objective-c code blocks with different color codes.
Fix some minor typos.

Fixes: #MOBILE-456

### CODE REVIEW

**Comment by ppult-gini** [comment]
@a-szotyori please also here a little bit of technical review.

File: doc/source/guides/common-tasks.rst [hunk 1]
```diff
@@ -108,6 +161,14 @@ The code example below shows how to send the error report to Gini.
...
+.. code-block:: swift
+    
+    // Assuming that `gini` is an instance of the `GiniSDK` facade class and
+    // `document` is an instance of the `GINIDocument` class as returned by `createDocumentWithFilename:fromImage:`.
+
+    let apiManager = gini.APIManager
+    let reportTask = apiManager?.reportErrorForDocument(document.documentId, summary: "short summary", description: "detailed description")
+
 Handling SDK errors
 ===================
```
Comment (katrobb) [inline doc/source/guides/common-tasks.rst]: All errors that occur (not occure) while executing a task...You can react to those errors by....

File: doc/source/guides/swift.rst [hunk 2]
```diff
@@ -4,21 +4,18 @@
...
-In order to use the Gini SDK in Swift code, you first need to add a bridge header file. A good tutorial how to add a
-bridge header file can be found at `Medium <https://medium.com/@stigi/swift-cocoapods-da09d8ba6dd2>`_.
+We assume you have already integrated the Gini SDK into your project. If not, take a look at our :ref:`integration guide <guide-getting-started>`.
 
-Inside the bridging header, you must import the Gini SDK by adding the following line:
+Module import
+===============
 
-.. code-block:: obj-c
+In order to use the Gini SDK in Swift code, you have to import the Gini iOS SDK module in every file you want to use it.
```
Comment (katrobb) [inline doc/source/guides/swift.rst]: ... you have to import the Gini SDK module in each file you want to use it in.

**Comment by katrobb** [comment]
Let's call it Gini SDK. Everywhere ;-)

**Pull request merged**